pipeline/filters/aws-metadata.md

```diff
@@ -60,42 +68,43 @@ $ bin/fluent-bit -c /PATH_TO_CONF_FILE/fluent-bit.conf
     Match *
 ```
 
-## EC2 Tags
+## EC2 tags
 
-EC2 Tags are a useful feature that enables you to label and organize your EC2 instances by creating custom-defined key-value pairs. These tags are commonly utilized for resource management, cost allocation, and automation. Consequently, including them in the Fluent Bit generated logs is almost essential.
+EC2 Tags let you to label and organize your EC2 instances by creating custom-defined key-value pairs. These tags are commonly utilized for resource management, cost allocation, and automation. Including them in the Fluent Bit generated logs is almost essential.
```

Comment:
Fix typo introduced by edit, avoid "utilized" for "used", and hyphenate compound term:

```suggestion
EC2 Tags let you label and organize your EC2 instances by creating custom-defined key-value pairs. These tags are commonly used for resource management, cost allocation, and automation. Including them in the Fluent Bit-generated logs is almost essential.
```